tests/Unit/DepreciationTest.php

```diff
@@ -38,8 +38,9 @@ public function testDepreciationAmount()
             ->laptopMbp()
             ->create(
                 [
-                    'category_id' => Category::factory()->assetLaptopCategory()->create(),
-                    'purchase_date' => now()->subDecade(),
+                    //not sure how this ever worked... do these need a category??
+                    //'category_id' => Category::factory()->assetLaptopCategory()->create(),
```

Comment:
I think we can remove this comment. I think turning on validation exceptions is causing this?